src/vaadin-rich-text-editor.html

```diff
@@ -288,9 +332,14 @@
                   alignCenter: 'align center',
                   alignRight: 'align right',
                   image: 'image',
+                  link: 'link',
                   blockquote: 'blockquote',
                   codeBlock: 'code block',
-                  clean: 'clean'
+                  clean: 'clean',
+                  enterLinkUrl: 'Enter link URL:',
```

Comment:
“Link address”

Reasons:
- The “Enter” verb, in the beginning, seems unnecessary. Start with “Link” so that users can more quickly identify that they clicked the right toolbar button.
- “URL” is a technical acronym which is most likely unknown to a regular user
- Titles should not have a semicolon or other punctuation at the end (a question mark is an exception)

Maybe the property name could also be something like `linkDialogTitle`? That leaves the exact wording more up to the user while being descriptive. It makes it an odd one in the mix, though, so I’m not sure what the best option would be. Using `linkAddress` would be in line with the others, but doesn’t seem descriptive enough.